client/src/components/Map/LeafletMap_Tour.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { MapContainer, Popup, TileLayer, Marker } from "react-leaflet";
+import styled from "styled-components";
+import "leaflet/dist/leaflet.css";
+
+import PropTypes from "prop-types";
+import { useLocation } from "react-router-dom";
+import { useQuery } from "react-query";
+import { getTourDetails } from "../../api/locations";
+
+import L from "leaflet";
+import icon from "leaflet/dist/images/marker-icon.png";
+import iconShadow from "leaflet/dist/images/marker-shadow.png";
+
+const Map = styled(MapContainer)`
+  height: 300px;
+  border-radius: 15px;
```

Comment:
Here I would probably not use hardcoded units such as px, if you use vh, vw or % the size will adapt to different screen sizes and makes the layout more flexible. :)

Reply:
Changed the mao from a hardcoded (px) height to vw :)